styles.css

```diff
@@ -0,0 +1,268 @@
+    color: var(--white);
+    padding: 2rem 5rem;
+    box-shadow: 0 0 2rem -0.1rem var(--gray-dominant);
+  }
+
+  .nav-header {
+    position: sticky;
+    top: 0;
+    z-index: 99999;
+}
+
+  .section-header {
+    margin: 3rem 0;
+  }
+
+  .right-icons {
+    display: flex;
+    gap: 2rem;
+  }
+
+  .btn-toggle {
+    cursor: pointer;
+  }
+  
+  footer {
+    min-width: 100vw;
+    padding: 1rem;
+    flex-direction: column;
+    gap: 1rem;
+    background: var(--pink-nav);
+    color: var(--white);
+    box-shadow: 0 0 1rem 0.2rem var(--gray-lighter);
+  }
+
+  .fab:hover {
+    color: var(--white);
+  }
+
+  /* width */
+::-webkit-scrollbar {
+  width: 1rem;
+}
+
+/* Track */
+::-webkit-scrollbar-track {
+  box-shadow: inset 0 0 5px var(--gray-lighter); 
+  border-radius: 5rem;
+}
+ 
+/* Handle */
+::-webkit-scrollbar-thumb {
+  background: var(--pink-mild);
+  border-radius: 5rem;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+  background: var(--pink-dominant); 
+}
+  
+  main {
+    grid-area: main;
+    overflow-y: auto;
+    overflow-x: hidden;
+    padding: 2rem 5rem;
+    margin-top: 2rem;
+  }
+
+  a {
+    text-decoration: none;
+    color:inherit;
+  }
+  a:hover {
+    color: var(--pink-light);
+    cursor: pointer;
+  }
+
+  ul {
+    list-style: none;
+    padding: 0 1rem;
+  }
+  .list {
+    display: flex;
+    flex-direction: column;
+    margin: auto;
+    gap: 1rem;
+  }
+
+  .to-top-btn{
+    position:absolute;
+    bottom:5rem;
+    right:5rem;
+  }
+
+  .ui-logo {
+    height: 5rem;
+    width: 5rem;
+    border-radius: 50%;
+  }
+
+.with-love  {
+  font-size: large;
+}
+
+.social-links {
+  padding-inline-start: 0px;
+}
+
+.list-item {
+  display: inline;
+}
+
+.icons {
+  font-size: 1.5rem;
+  padding: 0 0 0 2rem;
+}
+
+.icons-social {
+  font-size: 2.5rem;
+  padding: 0 0 0 2rem;
+}
+
+.span-imp {
+  font-weight: bolder;
+  color: var(--pink-nav);
+  text-shadow: -0.1rem 0 var(--white), 0 0.1rem var(--white), 0.1rem 0 var(--white), 0 -0.1rem var(--white),  0.2rem 0.2rem 0.6rem rgba(0,0,0,0.5); 
+}
+
+.hero-txt {
+  color: var(--white);
+  margin-top: 2rem;
+}
+
+.hero-image {
+  display: flex;
+  height: 64rem;
+  width: 100vw;
+  position: relative;
+  margin-left: -5rem;
+  z-index: -1;
+  margin-top: -26rem;
+}
+
+.btn-floating-gs {
+  border: 0.2rem solid var(--pink-dominant);
+  padding: 1rem;
+  border-radius: 2rem;
+  margin: 2rem 0;
+  background: var(--pink-dominant);
+  border: medium solid var(--pink-mild);
+  box-shadow: 0.2rem 0.2rem 1rem 0.1rem var(--gray-mild);
+  color: var(--white);
+  display: flex;
+  gap: 1rem;
+  font-size: 2rem;
+}
+
+.container-feature {
+  margin: 10rem 0 0 0;
+}
+
+.section-features {
+  display: flex;
+  flex-direction: column;
+  margin: 1rem 2rem;
+  color: var(--pink-dominant);
+  align-items: flex-start;
+  gap: 2rem;
+}
+
+iframe {
+  display: flex;
+  justify-content: flex-start;
+}
+
+@media screen and ( max-width:600px){
```

Comment:
I was told we are supposed to use em in media queries, so I'm suggesting it to you as well

Reply:
Will keep that in mind